### DISCUSSION TITLE
SmartTV

### DISCUSSION THREAD

**Discussion by imedjendoubi** [InfinityLoop1308/PipePipe#448] [Q&A]
Does it work with smart tvs?

Thank you!

**Comment by KonkaTV514** [comment]
This app in my konka Android TV(Android 9) works fine.

**Comment by InfinityLoop1308** [comment]
It should work fine as long as it can be installed :)

**Reply by imedjendoubi** [reply]
Working like a charm, but not really smart Tvs friendly

**Reply by InfinityLoop1308** [reply]
I don't have a smart TV so that is expected. Anyway if you have any suggestions, feel free to open an issue.

**Comment by KonkaTV514** [comment]
这应用在我的电视上用起来还行，就是缓存占用的空间有点多
